### PR TITLE
Add install instructions for all modules

### PR DESCRIPTION
**Pull request by kiview**
Adds hardcoded install instructions for all modules in the `installation` key of each module and for each implementing language.

After doing it by hand, I believe we can template the default cases and just hardcode the edge cases, to improve longterm maintenance. I will elaborate on how to do this later.

### CODE REVIEW

**Comment by kiview** [comment]
For all `core` modules, we have the following structure:

Java
```
<dependency>
    <groupId>org.testcontainers</groupId>
    <artifactId>$MODULE_NAME</artifactId>
    <version>$LATEST_JAVA_LIB_VERSION</version>
    <scope>test</scope>
</dependency>
```

Go
```
go get github.com/testcontainers/testcontainers-go/modules/$MODULE_NAME
```

.NET
```
dotnet add package Testcontainers.$MODULE_NAME --version $LATEST_DOTNET_LIB_VERSION
```

Node.js
```
npm install @testcontainers/$MODULE_NAME --save-dev
```

For `community` and `official` modules, it is not always straight forward to infer the coordinates (unless hosted in our repository, then the same pattern as above applies). 

The only exception is Go, which in case for modules not hosted by us will in most cases be:
```
go get github.com/$REPOSITORY_NAME
```

File: modules/dex/index.md [hunk 1]
```diff
@@ -13,6 +13,14 @@ docs:
...
       container.start();
       ```
+    installation: |
+      ```xml
+      <dependency>
+          <groupId>wf.garnier</groupId>
+          <artifactId>testcontainers-dex</artifactId>
+          <version>3.2.0</version>
+          <scope>test</scope>
+    </dependency>
```
Comment (mdelapenya) [inline modules/dex/index.md]: We need to close the 3 backtips here

**Review by mdelapenya — APPROVED** [review]
LGTM! SO much needed!

File: modules/aerospike/index.md [hunk 2]
```diff
@@ -14,6 +14,10 @@ docs:
         return err
       }
       ```
+    installation: |
+      ```bash
+      go get -u github.com/ajeetdsouza/testcontainers-aerospike-go@latest
```
Comment (mdelapenya) [inline modules/aerospike/index.md:19]:
```suggestion
      go get -u github.com/ajeetdsouza/testcontainers-aerospike-go
```

**Pull request merged**